lib/bundle/brew_dumper.rb

```diff
@@ -61,6 +56,18 @@ def expand_cask_requirements
 
     private
 
+    def self.formulae_info
+      @@formulae_info ||= begin
```

Comment:
Actually I mean it make more sense to cache the result after it is sorted based on dependency order. 

In the core, we prefer